nengo_loihi/builder/connection.py

```diff
@@ -589,7 +595,7 @@ def build_full_chip_connection(model, conn):  # noqa: C901
         decoder_block.compartment.configure_filter(tau_s, dt=model.dt)
         post_tau = model.decode_tau
 
-        target_axons = -np.ones(pre_obj.n_neurons, dtype=int)
+        target_axons = -np.ones(pre_obj.n_neurons, dtype=np.int32)
```

Comment:
Yeah, might as well change them all. I bet the `conv.py` ones snuck in there on one of the rebases, because I'm pretty sure I had changed all of them there originally. The allocators one I guess I just missed, and the test ones don't really matter because it's just a test but might as well change them too. In any case, good catches.